download-latest.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# This script works with a GitHub token to increase your request limit (for example, if using this script in a CI).
```

Comment:
```suggestion
# This script can optionally use a GitHub token to increase your request limit (for example, if using this script in a CI).
```

Slight reword to make it clear that using a GitHub token, while useful, is optional (I had a doubt from the initial reading)